docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -55,6 +55,14 @@ We have added 4 new features to the flamegraph visualization:
 
 {{< video-embed src="/media/docs/grafana/panels-visualizations/screen-recording-grafana-10.1-flamegraph-whatsnew.mp4" >}}
 
+### Visualizations and Widgets split
+
+<!-- Alexa Vargas -->
+
+_Experimental in some of our free tiers of Grafana._
+
+This experimental feature brings a clear distinction between two panel plugin types: visualizations and widgets. Now, easily add widgets like Text, News, and Annotation list, without selecting a data source. The plugins list and library panels are filtered based on widget or visualization type, providing a streamlined editing experience. Early access to this exciting update will be granted to some of our Cloud free tier users.
```

Comment:
Similar here "Early access to this exciting update will be granted to some of our Cloud free tier users". We usually say something like "To try X, enable the `Y` feature toggle."

Reply:
I applied the suggestions :)